applications/GSoC/AR-weekly-updates/week4.md

```diff
@@ -0,0 +1,17 @@
+# Week 4 (28th June - 4th July) 
+
+**Done**
+
+* Made changes in PR #114 and #115 after review by James. They should be merged after James gives one final review.
+* Added some items to checklist in issue #119 and set a date to show the MVP to the community.
+
+**Planned**
+
+* Present MVP to the community
+* Add documentation for the code
+* List feedback from the community and make changes
+
+**Blockers**
+
+Missed out on a bit of time due to side effects from the vaccine. Will be covering up remaining work this weekend.
```

Comment:
I'm glad you got the vaccine @anoura12!! Don't worry about catching up after illness! 💉 ❤️

Reply:
Thank you so much @KirstieJane !